bin/brew

```diff
@@ -72,28 +72,36 @@ do
   VAR_NEW="HOMEBREW_${VAR}"
   # Skip if existing HOMEBREW_* variable is set.
   [[ -n "${!VAR_NEW}" ]] && continue
-  export "$VAR_NEW"="${!VAR}"
+  export "${VAR_NEW}"="${!VAR}"
 done
 
 export HOMEBREW_BREW_FILE
 export HOMEBREW_PREFIX
 export HOMEBREW_REPOSITORY
 export HOMEBREW_LIBRARY
 
+export VISUAL
```

Comment:
Let's not add any new `export`s in here. Ignore them with a shellcheck ignore if needed.